toolbox/cluster/set_scale.sh

```diff
@@ -36,18 +35,30 @@ fi
 source ${THIS_DIR}/../_common.sh
 
 INSTANCE_TYPE=${1}
-ANSIBLE_OPTS="${ANSIBLE_OPTS} -e machineset_instance_type=${INSTANCE_TYPE}"
-
+shift 
+SCALE=${1}
 shift
+if [ $# -ne 0 ]; then
+    if [ "${1}" == "--base-machineset" ]; then
+        shift
+        BASE_MACHINESET=${1}
+        ANSIBLE_OPTS="${ANSIBLE_OPTS} -e base_machineset=${BASE_MACHINESET}"
+        shift
+        if [ $# -ne 0 && "${1}" == "--force" ]; then
```

Comment:
this `and` isn't valid, see there for examples https://stackoverflow.com/questions/13408493/an-and-operator-for-an-if-statement-in-bash using `[`

or replace `[` by `[[` (and closing)